spec/requests/locations/post_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 module Mutations
-  RSpec.describe "Posts" do
+  RSpec.describe "Posts", :vcr do
```

Comment:
what is `:vcr` ? how are you using it? I'm asking because I'm not sure and this might  be a learning moment for me. 

we use VCR's to capture 3rd party api calls and store the result in a YAML file, however I'm not seeing that here so I'm curious what and how thats used?

Reply:
We are calling MapQuest's API anytime a post request is made to add latitude and longitude to the location being created in the database.